msg/WStrings.msg

```diff
@@ -0,0 +1,10 @@
+wstring wstring_value
+#wstring wstring_value_default1 "Hello world!"
```

Comment:
Maybe add a TODO explaining why these are commented out.

Reply:
I will follow on these right away trying to enable all of them...